sql/hive/src/test/scala/org/apache/spark/sql/sources/BucketedWriteSuite.scala

```diff
@@ -169,19 +169,20 @@ class BucketedWriteSuite extends QueryTest with SQLTestUtils with TestHiveSingle
     }
   }
 
-  test("write bucketed data with the overlapping bucketBy and partitionBy columns") {
-    intercept[AnalysisException](df.write
+  test("write bucketed data with the overlapping bucketBy/sortBy and partitionBy columns") {
```

Comment:
Not related to this PR, but I think we should move most test cases to sql packages. Let me try to do it. Only orc formats are hive only.